tools/topology/topology2/platform/intel/sdw-amp-feedback.conf

```diff
@@ -0,0 +1,64 @@
+Object.Dai {
+	ALH."515" {
+		id 		$SDW_AMP_IN_BE_ID
+		direction	"capture"
+		name		$SDW_SPK_IN_STREAM
+		default_hw_conf_id	1
+		rate			48000
+		channels		2
+
+		Object.Base.hw_config."ALH515" {
+			id	1
+		}
+	}
+}
+
+Object.Pipeline {
+	passthrough-capture.30 {
+		index 30
+		Object.Widget.pipeline.1.stream_name	"SPK IN stream"
+
+		Object.Widget.copier.1.stream_name	"amp feedback"
+		Object.Widget.copier.1.Object.Base.audio_format.1 {
+			# 32 -> 16 bits conversion is done here,
+			# so in_bit_depth is 32 (and out_bit_depth is 16).
```

Comment:
comment is very cryptic, what are you trying to say here?

Reply:
> comment is very cryptic, what are you trying to say here?

That was copy paste mistake. I will remove the comment.